### DISCUSSION TITLE
Add ability to send notifications to topics using Magicbell admin console

### DISCUSSION THREAD

**Discussion by RuslanHryn** [magicbell/community#216] [General]
Please add the ability to send notifications to topics using Magicbell admin console. Currently we can send notifications only to users using Magicbell admin console

**Comment by smeijer** [comment]
Hi Ruslan,

Thanks for the request. Topics are a powerful feature, which we plan to add to our new compose page. Unfortunately, it's not as simple as adding a "topic field", as we'd need to support lookup/searching, while topics make most sense in backend integrations, and less when manually sending broadcasts.

Please try our [CLI](https://npmjs.com/@magicbell/cli) if you're using the compose page for testing purposes.
